draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -513,11 +519,27 @@ Initiator                             Responder
                 using IKE_INTERMEDIATE exchange MUST be treated as protocol error by both initiator and responder.
                 </t>
 
+
+                <figure><artwork align="center" ><![CDATA[
+Initiator                          Responder
+---------------------------------------------------------------------
+HDR, SAi1(.. AKE*...), KEi1, Ni,
+N(INTERMEDIATE_EXCHANGE_SUPPORTED)    --->
+                                   HDR, SAr1(.. AKE*...), KEr1, Nr,
+                                   [CERTREQ],
+                           <---    N(INTERMEDIATE_EXCHANGE_SUPPORTED)
+                ]]></artwork></figure>
+
                 <t> The responder performs negotiation using standard IKEv2 procedure described in Section 3.3 of <xref target="RFC7296"/>.
                 However, for the Additional Key Exchange types the responder's choice MUST NOT contain equal transform IDs (apart from NONE), 
                 and the ID selected for Transform Type 4 MUST NOT appear in any of Additional Key Exchange transforms.
                 In other words, all selected key exchange methods must be different.
                 </t>
+
+                <t> If the responder selected NONE for some Additional Key Exchange types (provided they were proposed by the initiator), 
+                then the corresponding IKE_INTERMEDIATE exchanges should not take place. The IKE_INTERMEDIATE exchanges
+                MUST only be performed for Additional Key Exchange types cintaining non-NONE responders choices.
```

Comment:
```suggestion
                MUST only be performed for Additional Key Exchange types containing non-NONE responders choices.
```
```suggestion
                MUST only be performed for Additional Key Exchange types containing non-NONE responders choices.
```

Reply:
I agree, thank you for catching this typo.